CHANGELOG.md

```diff
@@ -65,6 +65,7 @@
 * [#5283](https://github.com/bbatsov/rubocop/issues/5283): Change file path output by `Formatter::JSONFormatter` from relative path to smart path. ([@koic][])
 * `Style/SafeNavigation` will now register an offense for methods that `nil` responds to. ([@rrosenblum][])
 * [#5542](https://github.com/bbatsov/rubocop/pull/5542): Exclude `.git/` by default. ([@pocke][])
+* Tell Read the Docs to build downloadable docs. ([@eostrom][])
```

Comment:
I guess you forgot to add yourself to the bottom of the changelog.

Reply:
Aha. Fixed.